smithy-codegen-core/src/test/resources/software/amazon/smithy/codegen/core/definitions.txt

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
If these files are valid JSON, which it looks like, they can be `.json` files instead of `.txt`